### PR TITLE
Various documentation fixes

### PR DESCRIPTION
**Pull request by csoneson**
This PR contains some small suggestions to improve the clarity and consistency of the documentation (and some typo fixes). Please let me know if there is anything else that should be done/rendered locally from these files before pushing, that I may have missed.

### CODE REVIEW

File: src/ez_zarr/hcs_wrappers.py [hunk 1]
```diff
@@ -1319,24 +1319,24 @@ class FractalZarrSet:
...
         a three-dimensional maximum intensity projection derived from it.
 
         Parameters:
-            path (str): Path containing the ome-zarr fileset(s).
+            path (str): Path containing the OME-Zarr fileset(s).
             name (str): Optional name for the experiment.
         
         Examples:
-            Get an object corresponding to a set of .zarr's.
+            Get an object corresponding to a set of .zarrs.
```
Comment (silvbarb) [inline src/ez_zarr/hcs_wrappers.py:1333]: File extension is '.zarr', so how about we replace it with ".zarr"'s?

File: src/ez_zarr/hcs_wrappers.py [hunk 2]
```diff
@@ -1319,24 +1319,24 @@ class FractalZarrSet:
...
 
             >>> from ez_zarr import hcs_wrappers
             >>> plate_set = hcs_wrappers.FractalZarrSet('path/to/zarrs')
             >>> plate_set
 
-            This will print information on the zarrs.
+            This will print information on the .zarrs.
```
Comment (silvbarb) [inline src/ez_zarr/hcs_wrappers.py:1339]: What convention do we want to stick to? Should we keep the singular (.zarr) as this is the file extension?

**Comment by silvbarb** [comment]
Looks great to me, Thanks Charlotte! 

The only thing I am not sure is which convention we want to stick to when it comes to '.zarr', as this is the file extension of the OME-Zarr produced by fractal (and we do not have .zarrs, but multiple .zarr files). 
What do you think?

File: src/ez_zarr/hcs_wrappers.py (hunk 1 above)
Comment (csoneson) [reply src/ez_zarr/hcs_wrappers.py:1333]: Sure (I don't really have strong opinions here) - but I would go without the genitive apostrophe. So either .zarrs, `.zarr`s, or ".zarr"s?

**Comment by csoneson** [comment]
Yeah, not sure - one option could be to put the `.zarr` in  fixed-width case and then add the s, i.e. `.zarr`s (I've sometimes done this when referring to `data.frame`s etc.)

**Pull request closed without merging**